src/app/login/page.tsx

```
@@ -0,0 +1,236 @@
+                default:
+                  router.push("/dashboard");
+              }
+              toast.success("Login successful");
+            }
+          }
+        },
+        onError: (error) => {
+          toast.error(error.message || "Login failed. Please try again.");
+        },
+      });
+    } catch (err) {
```

ESLint

[error] 'err' is defined but never used.

(@typescript-eslint/no-unused-vars)